infra/gcp/clusters/projects/k8s-infra-prow-build/prow-build/main.tf

```diff
@@ -117,7 +117,7 @@ module "prow_build_nodepool_n1_highmem_8_maxiops" {
   name            = "pool4"
   initial_count   = 1
   min_count       = 1
-  max_count       = 50
+  max_count       = 80
```

Comment:
Reminder that this limit is zonal. The new total capacity limit will be 3x80 = 240

Reply:
I'll figure out whether we need to bump quotas to support this in https://github.com/kubernetes/k8s.io/issues/1703 but we're definitely not bumping into quota at 50, a 60% increase seems reasonable